[process] Removing STF from the environment (#57)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_removing-stf-from-the-openshift-environment.adoc

// The file name and the ID are based on the module title. For example:
// * file name: proc_doing-procedure-a.adoc
// * ID: [id='proc_doing-procedure-a_{context}']
// * Title: = Doing procedure A
//
// The ID is used as an anchor for linking to the module. Avoid changing
// it after the module has been published to ensure existing links are not
// broken.
//
// The `context` attribute enables module reuse. Every module's ID includes
// {context}, which ensures that the module has a unique ID even if it is
// reused multiple times in a guide.
//
// Start the title with a verb, such as Creating or Create. See also
// _Wording of headings_ in _The IBM Style Guide_.
[id='removing-stf-from-the-openshift-environment_{context}']
= Removing {ProjectShort} from the {OpenShiftShort} environment

To remove {ProjectShort} from an {OpenShiftShort} environment, complete the following tasks:

. Delete the namespace. For more information, see <<removing-the-namespace>>.
. Delete the OperatorSource. For more information, see <<removing-the-operatorsource>>.

[id='removing-the-namespace']
== Removing the namespace

> Most of the deployed resources for {ProjectShort} can be removed from {OpenShiftShort} by deleting the namespace. To delete a namespace run the command:

Reviewer: ```suggestion
To remove most of the deployed resources for {ProjectShort} from {OpenShiftShort}, delete the namespace:
```

Reviewer: More instructional, replacing "by deleting" with just "delete"

[source,bash]
----
oc delete project service-telemetry
----

> You can validate that all the resources have been removed from the namespace by running `oc get all`.

Reviewer: ```suggestion
Validate that the resources have been removed from the namespace:
```

Reviewer: Separating instruction from command. It's such a short step that it looks better this way, we avoid repetition and avoid using "by running"

[source,bash]
----
$ oc get all
No resources found.
----

[id='removing-the-operatorsource']
== Removing the OperatorSource

> If future installations of {Project} are no longer expected, the OperatorSource can be removed. Removing the OperatorSource will result in the PackageManifests

Reviewer: ```suggestion
If you do not expect to install {Project} again, remove the OperatorSource. When you remove the OperatorSource, PackageManifests related to {ProjectShort} are removed from the Operator Lifecycle Manager catalog.
```

Reviewer: So the PackageManifests are automatically removed? You don't have to run a command or anything, correct?

Author: Correct. When you remove the `CatalogSource` or `OperatorSource` the `PackageManifests` for that source are automatically removed.

related to {ProjectShort} being removed from the Operator Lifecycle Manager catalog.

[source,bash]
----
$ oc delete --namespace=openshift-marketplace operatorsource infrawatch-operators
operatorsource.operators.coreos.com "infrawatch-operators" deleted
----

> You can validate that the {ProjectShort} PackageManifests are removed from the platform by running `oc get packagemanifests | grep InfraWatch`, which is expected to return no result.

Reviewer: ```suggestion
Validate that the {ProjectShort} PackageManifests are removed from the platform. If successful, the following command returns no result: 
```
